src/redux/books/books.js

```diff
@@ -11,12 +12,16 @@ export const removeBookFunction = (id) => (dispatch) => dispatch({
 });
 
 // reducer function
-const bookReducer = (state = [], action) => {
+const bookReducer = (state = data, action) => {
   switch (action.type) {
     case ADD_BOOK:
-      return [...state, action.payload];
+      return [...state, {
+        id: action.payload.id,
+        title: action.payload.author,
+        category: action.payload.category,
+      }];
```

Comment:
- [ ] When I add a book, the author displays but the book title doesn't display. In the bookReducer, when you are adding a book, you forgot to include the author key. Also, the title key displays the author instead of the title. Kindly fi this error.

```javascript
title: action.payload.title,
author: action.payload.author
```